backend/substrapp/tasks/tasks.py

```diff
@@ -1100,3 +1101,15 @@ def on_compute_plan(channel_name, compute_plan):
     # Remove intermediary models
     if model_keys:
         remove_intermediary_models(model_keys)
+
+
+@app.task(ignore_result=True)
+def clean_old_images_task(max_duration):
+    algo_image_names = fetch_old_algo_image_names(max_duration)
+    for algo_image_name in algo_image_names:
+        remove_image(algo_image_name)
+
+
+@app.task(ignore_result=True)
+def garbage_collector_task():
+    garbage_collector()
```

Comment:
Make it more clear what this is?

```suggestion
def docker_registry_garbage_collector_task():
    docker_registry_garbage_collector()
```